src/app/components/simple-editable-list/SimpleEditableList.jsx

```diff
@@ -67,7 +67,7 @@ export default class SimpleEditableList extends Component {
                 ) : null}
                 <button
                     type="button"
-                    className={"btn btn--link " + (this.props.fields.length ? "margin-top--1" : "")}
+                    className={"btn btn--link " + (this.props.fields?.length ? "margin-top--1" : "")}
```

Comment:
I guess it does work, interesting. 

So is it like if there is a fields property check it's length?

Reply:
Hi Josh it is https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining